inc/class-thumbnails.php

```diff
@@ -174,6 +184,15 @@ public function get_data( $post, $url ) {
 			$id = strpos( $id, '?' ) !== false ? substr( $id, 0, strpos( $id, '?' ) ) : $id;
 			$thumbnail = get_post_meta( $post->ID, 'embed_privacy_thumbnail_youtube_' . $id, true );
 		}
+		else if ( strpos( $url, 'slideshare.net' ) !== false ) {
```

Comment:
Please resort to begin with slideshare in the if statements (should always be in alphabetical ascending order)

Reply:
Done